src/content/docs/browser/single-page-app-monitoring/get-started/spa-v2.mdx

```diff
@@ -0,0 +1,106 @@
+
+<Callout variant="important">
+    This release is currently in optional trial mode for any browser customer who wants to explore the new features and provide your feedback. The current SPA experience will be gradually phased out as we prepare for a full rollout to all SPA monitoring customers.
+    </Callout>
+
+## Painpoints addressed
+
+* Unusable latest versions: Frequent conflicts with third-party libraries and unreliable interaction capture plagued the existing agent, often rendering the latest version unusable.
+* Patchwork solutions: Addressing one issue with SPA patches often introduced another, creating a frustrating cycle of fixes and regressions.
+* Third-party library conflicts: Global wrapping, particularly around Promises, often disrupted code functionality due to conflicts with other libraries.
+* Performance bottlenecks: Conflicts with code using timers, RAF, and promise chaining led to performance issues, ranging from slowdowns to freezes.
+
+The Pro+SPAv2 agent is designed to eliminate these problems and provide a significantly improved monitoring experience. Key changes include:
+
+* Unwrapped execution: By not wrapping core globals, Pro+SPAv2 unleashes execution speed boosts for your SPAs.
+* Aligned with SoftNav heuristics: The new agent adopts Google Chrome's soft navigation heuristics, providing more accurate interaction tracking and improved alignment with browser behavior.
+* Simplified interaction determination: Interactions are now defined as UI event (click, keydown, submit) -> route change -> DOM modification, offering a clearer and more efficient capture approach.
+Focus on key metrics: While reported data remains largely unchanged, the new agent no longer tracks JS execution or callback duration within interactions, streamlining the reported information.
+
+## Impact and user experience
+
+Changes to the SPA monitoring include:
+
+* Reduced interaction durations: Expect significantly shorter interaction durations, particularly for route changes. Initial page load interactions should remain stable.
+* Potential event disassociation: You might observe previously associated `AjaxRequest` and `JavascriptError` events becoming disassociated from interactions, reflecting the focus on UI-driven interactions.
+* API compatibility: The current SPA API remains entirely compatible, ensuring seamless integration with your existing code.
+* Enhanced control: The new [`newrelic.interaction({ waitForEnd: true })`](/docs/browser/new-relic-browser/browser-apis/interaction/) API function empowers you to customize interaction completion behavior if needed.
+* Deprecated `createTracer`: While still functional, the `createTracer` function is deprecated as it no longer keeps interactions open or times callbacks. Our data indicates minimal usage of this function.
```

Comment:
Should this go under API compatibility as a sub-bullet?

Reply:
They should note `BrowserTiming` events are also no longer created when using `createTracer` with soft nav